index.md

```diff
@@ -1,6 +1,115 @@
 ---
 layout: default
-title: Home
+title: Introducing Data Commons
```

Comment:
Just wondering if this will make Data Commons sound forever "new"

Reply:
Hm, good point. I'll update it in a followup